### PR TITLE
Fixes image rows on Designer to collapse to column on resize

### PR DESCRIPTION
**Pull request by JoeButler7**
### What does this PR do?
Changes the image rows on Designer so that on large and xlarge window sizes it will overhang background and on smaller window sizes row will collapse into a column

Requesting review from @jcfilben

### CODE REVIEW

File: src/pages/Designer/Designer.js [hunk 1]
```diff
@@ -117,55 +117,111 @@ const Designer = () => (
...
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Stack>
+              <Box
+                background="YellowBackground"
+                height="600px"
+                margin={{
+                  top: 'xlarge',
+                }}
+              ></Box>
```
Comment (ShimiSun) [inline src/pages/Designer/Designer.js:130]:
```suggestion
              />
```

File: src/pages/Designer/Designer.js [hunk 2]
```diff
@@ -117,55 +117,111 @@ const Designer = () => (
           <BarChart size="large" color="TabularGreen" />
         </Box>
       </Box>
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Stack>
+              <Box
+                background="YellowBackground"
+                height="600px"
```
Comment (ShimiSun) [inline src/pages/Designer/Designer.js:126]: hard coding this box will be tricky on responsive behavior when the tiles need to be set in a column.

File: src/pages/Designer/Designer.js [hunk 3]
```diff
@@ -117,55 +117,111 @@ const Designer = () => (
...
+                  top: 'large',
+                  horizontal: 'xlarge',
+                  bottom: 'medium',
+                }}
+              >
+                <Heading size="large" margin={{ top: 'xlarge' }}>
+                  What is the Designer?
+                </Heading>
+                <Box width="large" margin={{ bottom: 'medim', top: 'none' }}>
+                  <Paragraph fill={true} size="xxlarge">
```
Comment (ShimiSun) [inline src/pages/Designer/Designer.js:142]:
```suggestion
                  <Paragraph fill size="xxlarge">
```

File: src/pages/Designer/Designer.js [hunk 4]
```diff
@@ -117,55 +117,111 @@ const Designer = () => (
           <BarChart size="large" color="TabularGreen" />
         </Box>
       </Box>
+      <ResponsiveContext.Consumer>
+        {(responsive) =>
+          responsive === 'large' || responsive === 'xlarge' ? (
+            <Stack>
```
Comment (ShimiSun) [inline src/pages/Designer/Designer.js:123]: I'm wondering if should use Stack at all or just use a Box with background and place the context inside. And in order to achieve the offset of the tiles with the background, you can use a negative margin instead.
If you feel like it is too much work at this point, we can postpone this refactoring to after the showcase.

**Comment by JoeButler7** [comment]
### Updates: 
Removed stack in favor of negative margin. Also resolved merge conflicts

**Pull request merged**